.ci/releasenotes-gen.sh

```diff
@@ -54,7 +54,7 @@ cd .ci-temp
 java -jar contribution/releasenotes-builder/target/releasenotes-builder-1.0-all.jar \
         -localRepoPath checkstyle -remoteRepoPath checkstyle/checkstyle \
         -startRef "$LATEST_RELEASE_TAG" -releaseNumber "$CS_RELEASE_VERSION" \
-        -githubAuthToken "$READ_ONLY_TOKEN" -generateAll -publishXdoc
+        -githubAuthToken "$READ_ONLY_TOKEN" -generateAll
```

Comment:
I am confused what is the difference between this (releasenotes-gen.sh) and `releasenotes-gen-xdoc-push.sh`? Based on name, I would expect push to call the other, but just have extra code to do a push.

Reply:
old script is for CI to detect missed label and generate sdoc content for manual copy-paste during release

second script is planned to be executed during release process, without any extra stuff
![image](https://user-images.githubusercontent.com/812984/178150398-65a39f21-e86a-4562-9d26-377817d1e146.png)

but there might be room to optimize something.